pyproject.toml

```diff
@@ -82,7 +82,7 @@ pyinstaller-versionfile = "^2.0.0"
 tomli = ">=1.2,<3.0"
 
 [tool.poetry.extras]
-docs = [ "sphinx", "sphinx-rtd-theme", "sphinx-copybutton", "myst-nb" ]
+docs = [ "sphinx", "sphinx-rtd-theme", "sphinx-copybutton", "myst-nb", "furo" ]
```

Comment:
Can we remove sphinx-rtd-theme from here?

Reply:
fixed